packages/inventory/src/InventoryList.js

```diff
@@ -52,9 +54,9 @@ class ContextInventoryList extends React.Component {
 
     componentDidUpdate(prevProps) {
         const { items, hasItems, sortBy } = this.props;
-        if (hasItems && (JSON.stringify(items) !== JSON.stringify(prevProps.items))) {
+        if (hasItems && (isEqual(items, prevProps.items))) {
```

Comment:
These should be negated
```
if (hasItems && (!isEqual(items, prevProps.items))) {
```
} else if (!hasItems && !isEqual(prevProps.sortBy, sortBy)) {
```

```

Reply:
Oh...good catch!